### PR TITLE
Adding three new steps for job hunting

### PR DESCRIPTION
**Pull request by mahmoodc**
These steps will cover accessibility steps for cd advice, covering letter and interview advice pages.

### CODE REVIEW

File: cypress/integration/happy-path-with-accessibility.js [hunk 1]
```diff
@@ -146,9 +146,27 @@ describe('Get help to retrain smoke test', function() {
...
 
-  it('should allow me to find local schemes and offers', function() {
+  it('should allow me to get CV advice', function() {
     cy.go('back')
-    cy.contains('Show me local offers').click()
+    cy.contains('Get CV advice').click()
+    checkAccessibility();
+  });
+
+  it('should allow me to get cover letter advice', function() {
```
Comment (cwrw) [inline cypress/integration/happy-path-with-accessibility.js]: the cover letter advice step is repeated :)

File: cypress/integration/happy-path-with-accessibility.js [hunk 2]
```diff
@@ -146,9 +146,27 @@ describe('Get help to retrain smoke test', function() {
     checkAccessibility();
   });
 
-  it('should allow me to find local schemes and offers', function() {
```
Comment (cwrw) [inline cypress/integration/happy-path-with-accessibility.js]: could we keep the local schemes step and add a new cv advice step

File: cypress/integration/happy-path-with-accessibility.js (hunk 1 above)
Comment (mahmoodc) [reply cypress/integration/happy-path-with-accessibility.js]: deleted.

File: cypress/integration/happy-path-with-accessibility.js (hunk 2 above)
Comment (mahmoodc) [reply cypress/integration/happy-path-with-accessibility.js]: oops. fixed now.

File: cypress/integration/happy-path-with-accessibility.js [hunk 3]
```diff
@@ -146,6 +146,24 @@ describe('Get help to retrain smoke test', function() {
...
+  });
+
+  it('should allow me to get cover letter advice', function() {
+    cy.go('back')
+    cy.contains('Get cover letter advice').click()
+    checkAccessibility();
+  });
+  
+   it('should allow me to get interview advice', function() {
+    cy.go('back')
```
Comment (cwrw) [inline cypress/integration/happy-path-with-accessibility.js]: could you add the missing semicolons on each step (I think we missed this on the last pr as well)

Comment (mahmoodc) [reply cypress/integration/happy-path-with-accessibility.js]: We don't actually need them in cypress but I've added as per your request

**Pull request merged**